compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -60,6 +61,8 @@ struct ROCmOptions {
   int wavesPerEu = 0;
   std::string enableROCMUkernels = "none";
   bool legacySync = true;
+  // Optional pass plugins
+  std::vector<std::string> passPlugins;
```

Comment:
Use SmallVector to follow the llvm guidelines: https://llvm.org/docs/CodingStandards.html#c-standard-library